src/rez/package_test.py

```diff
@@ -103,8 +109,23 @@ def get_package(self):
             return self.package
 
         if self.use_current_env:
-            pass
+            # get package from current context, or return None
+            current_context = ResolvedContext.get_current()
+            if current_context is None:
+                return None
```

Comment:
No, because this results in tests getting skipped (see L341), not an error. Having said that, I see an almost-error on L543 that needs fixing, will do that now.

Reply:
ps - will make the change just mentioned in the second PR, it's minor.